wiki/LearnInPublic.md

```diff
@@ -6,53 +6,53 @@
 
 <div style="padding: 1rem; background-color: #F5F5F5;">
 
-# เรียนรู้แบบเปิดเผย
+# เรียนแล้วแชร์
 
 ## วิธีที่เร็วที่สุดในการเรียนรู้
 
-ถ้ามันจะมีกฎเหล็กสักข้อ กฎข้อนี้แหละสำคัญที่สุด จึงมาก่อนข้ออื่นๆ กฎข้ออื่นๆ ก็ต่อยอดมาจากกฎข้อนี้ทั้งนั้น
+ถ้ามันจะมีกฎเหล็กสักข้อ ก็คงเป็นข้อนี้แหละที่สำคัญที่สุด กฎข้อนี้เลยมาก่อนข้ออื่น ๆ ที่ก็ต่อยอดมาจากกฎข้อนี้ทั้งนั้น

```

Comment:
I recommend not adding space before ๆ as if the text has to be wrapped, the browser will sometimes break the ๆ to the next line which kills the layout.

(We might add a space before all ๆ characters when rendering the article in our website code.)

Reply:
okie dokie